Calibration/HcalAlCaRecoProducers/test/AlCaLowPUHBHEMuonFilter_cfg.py

```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("AlCaLowPUHBHEMuon")
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']
+
+process.load("RecoLocalCalo.EcalRecAlgos.EcalSeverityLevelESProducer_cfi")
+process.load("Configuration.StandardSequences.AlCaRecoStreams_cff")
+process.load("Configuration.EventContent.AlCaRecoOutput_cff")
+
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(10) )
+process.source = cms.Source("PoolSource",
+                            fileNames = cms.untracked.vstring(
+        'file:/eos/user/s/shghosh/FORSUNANDASIR/F256E27E-0930-E811-BE1A-FA163EBF1F42.root',
```

Comment:
@bsunanda this does not work (was it ever tested ?). Please replace "file:/eos" with "/store"

Reply:
@fabiocos Yes - it was tested with a file on the eos using the file in the lxplus. I can restore the file accessed through xrootd. But then one has to go through all testing and signing phases. Please suggest.

Reply:
@fabiocos Please let me know if you want the example cfg file tone updated for integration.